catalog.bom

```diff
@@ -1,127 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
No need for this header - the files are Apache licensed, not licensed to the ASF.